.gitignore

```diff
@@ -1,6 +1,7 @@
 /target
-.cargo
-.vscode/
+/.cargo
+/.vscode
+/.idea
```

Comment:
these things belong in your global gitignore file (see https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
reverted